src/main/java/eu/europa/ted/efx/sdk0/v6/entity/SdkCodelist06.java

```diff
@@ -14,6 +15,6 @@
 public class SdkCodelist06 extends SdkCodelist {
 
     public SdkCodelist06(String codelistId, String codelistVersion, List<String> codes) {
-        super(codelistId, codelistVersion, codes);
+        super(codelistId, codelistVersion, codes, Optional.empty());
```

Comment:
This cannot work, where is the parent id supposed to come from?

Reply:
WE do not care. The unsupported SDK will be removed in a separate PR.

Reply:
ah ok